test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,78 @@
+const agent = require('superagent');
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('First Api Tests', () => {
+
+    it('Consume GET Service', async () => {
+        const response = await agent.get('https://httpbin.org/ip');
+      
+        expect(response.status).to.equal(statusCode.OK);
+        expect(response.body).to.have.property('origin');
+      });
+
+    it('Consume GET Service with query parameters', async () => {
+        const query = {
+          name: 'John',
+          age: '31',
+          city: 'New York'
+        };
+      
+        const response = await agent.get('https://httpbin.org/get').query(query);
```

Comment:
Put the base url ('https://httpbin.org') on  a variable, so that you can re-use it and chang it easily

Reply:
I think its more complicated, because you need to concatenate the gets, puts, headers etc. But,  How did you do it?